BinaryEncoding.md

```diff
@@ -59,6 +59,32 @@ A single-byte unsigned integer indicating a [value type](AstSemantics.md#types).
 * `3` indicating type `f32` 
 * `4` indicating type `f64`
 
+### definition_kind
```

Comment:
Nit: How about `export_kind` or `external_kind`? It's an annotation on imports/exports, not definitions, and only includes things that can actually be imported/exported.